### PR TITLE
friends app done

### PR DESCRIPTION
**Pull request by Roka20012**
Friends App

[Demo](https://roka20012.github.io/friends-app/) |
[Code base](https://github.com/Roka20012/kottans_frontend/tree/master/practical_task/friends-app/project)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by Zakkarat** [comment]
Good job! :) 
But It seems I found a small miss, when there are not enough people on the page, left block does not seem to display properly. It's not big, but it will be great, if you will fix it.

**Comment by Roka20012** [comment]
@Zakkarat Thanks) Can you make a screenshot for me?

**Comment by Roka20012** [comment]
This is another fix)Do not pay attention :D

**Comment by Zakkarat** [comment]
> @Zakkarat Thanks) Can you make a screenshot for me?

Sure! Also age looks awkward, when phone number is underneath, It is hard for user to understand what it is exactly, could we do something like `Age: 36`? 
screen

**Comment by Roka20012** [comment]
Fix it)

**Comment by Temu4** [comment]
> Code base

It links to Popup)

**Comment by Roka20012** [comment]
@Temu4  Lol, fixed :smile:

File: submissions/Roka20012/friends-app/style.css [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+    width: 500px;
+    flex-wrap: wrap;
+  }
+  input[type="text"] {
+    width: 100px;
+    background-color: #fff;
+  }
+  input[type="text"]:focus {
+    width: 200px;
+    background-color: #fff;
```
Comment (Temu4) [inline submissions/Roka20012/friends-app/style.css]: In general, nice-looking app.
But, I think, little wider input field for mobile screens will be better.

**Comment by Oodmincheg** [comment]
Hello, guys. Nice collaboration. @Roka20012 I'm not an expert in UI but I find you menu toggler a little bit confusing. It gets some time for me to understand where is menu and I read you discussion before. So it can be more longer time for new user of your app. I think that is "hamburger button menu" it's kinda standard nowadays. But. I'm really not expert. What do you think? 
And it's definitely good idea to make your search case insensitive :)

File: submissions/Roka20012/friends-app/style.css (hunk 1 above)
Comment (Roka20012) [reply submissions/Roka20012/friends-app/style.css]: No problem fixed)

**Comment by Roka20012** [comment]
@Oodmincheg Hello, already done:)

**Comment by Oodmincheg** [comment]
Prob

> @Oodmincheg Hello, already done:)

Hello, man. There is some problems with search. If I try find "p" charter for example :)

**Comment by Roka20012** [comment]
@Oodmincheg hi) I made some changes please check)

**Comment by Roka20012** [comment]
@Oodmincheg i made some changes)

**Pull request merged**